.github/workflows/angular.yml

```diff
@@ -26,4 +26,4 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm install
     - run: npm run build
-    #- run: npm run test
+    - run: npm run test
```

Comment:
Re-enabling tests during PR validation

Reply:
I also noticed that the workflow uses `actions/checkout@v2` while `actions/checkout@v3` is stable.
Should that also be updated?

Reply:
Good point